### PR TITLE
Change notebook's style

### PR DESCRIPTION
**Pull request by Qti3e**
propel

### CODE REVIEW

**Comment by Qti3e** [comment]
The only todo on this PR is about setting colors using SCSS variables :thinking:

File: website/main.scss [hunk 1]
```diff
@@ -487,24 +487,40 @@ svg {
...
+  border-radius: 90px;
+  height: 50px;
+  width: 50px;
+  margin-right: 5px;
 }
-
 .most-recent li a,
 .most-recent li p {
-  color: #fff;
+  color: #373737;
```
Comment (ry) [inline website/main.scss]: Can you move colors to scss variables and put them in `syntax_light.scss` and `syntax_dark.scss`

File: website/main.scss [hunk 2]
```diff
@@ -487,24 +487,40 @@ svg {
...
 }
 
 .docindex a:hover,
 .docindex a:focus,
 .docindex a:active {
-  color: rgb(31, 119, 180);
+  color: #1f77b4;
```
Comment (ry) [inline website/main.scss]: scss variable

File: website/main.scss [hunk 3]
```diff
@@ -487,24 +487,40 @@ svg {
...
   text-decoration: underline;
-  background-color: $base1;
+  background-color: #ddd;
```
Comment (ry) [inline website/main.scss]: scss variable

File: website/main.scss [hunk 4]
```diff
@@ -487,24 +487,40 @@ svg {
...
+  box-shadow: -3px 2px 17px #6b767b;
+  width: 100%;
+  padding: 5px;
+  border-radius: 5px;
+  transition: background-color 0.5s ease;
+}
+.most-recent .blurb{
+  display: flex;
+  height: 55px;
+  border-bottom: 2px solid #ccc;
```
Comment (ry) [inline website/main.scss]: scss variable

File: website/main.scss [hunk 5]
```diff
@@ -487,24 +487,40 @@ svg {
   list-style-type: none;
   flex-wrap: wrap;
 }
-
+.most-recent li:hover{
+  background-color: #dee1e2;
```
Comment (ry) [inline website/main.scss]: scss variable

**Comment by Qti3e** [comment]
I have trouble naming scss variables, but I can reuse variables you already defined.

Use case | Hex Color | Suggested name
--- | --- | ---
Item on hover color| #dee1e2 | $base2
Item box shadow color | #6b767b | $base00
Header bottom border color | #ccc | $base01
Item text color |  #373737 | $base03

**Pull request closed without merging**